client/my-sites/earn/home.tsx

```diff
@@ -183,52 +182,6 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		};
 	};
 
-	/**
-	 * Return the content to display in the Referrals card based on the current plan.
-	 *
-	 * @returns {object} Object with props to render a PromoCard.
-	 */
-	const getReferralsCard = () => {
```

Comment:
Is there any reason to leave this function around for now? Maybe we plan to add the referral card back for some users/sites in the near future?

Reply:
I don't we'll be doing that, but if we do, we can reference this PR. The pre-commit checks won't let me leave the dead code in.